lib/permutat.g

```diff
@@ -763,6 +763,13 @@ InstallMethod( DistancePerms, "for general permutations",
         function(x,y)
     return NrMovedPoints(x/y); end);
 
+#############################################################################
+##
+#V  PERM_INVERSE_THRESHOLD . . . . cut off for when inverses are computed 
+##                                 eagerly
+##
+
+BIND_GLOBAL( "PERM_INVERSE_THRESHOLD", 10000 );
```

Comment:
Document the unit of this threshold? I am guessing it's the degree of the permutation, and we'll automatically compute inverses for anything with smaller degree?